packages/blockchain-wallet-v4-frontend/src/modals/IdentityVerification/index.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { compose, bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { keys } from 'ramda'
+import { FormattedMessage } from 'react-intl'
+
+import { actions } from 'data'
+import { STEPS } from 'data/components/identityVerification/model'
+import { getData } from './selectors'
+import modalEnhancer from 'providers/ModalEnhancer'
+import media from 'services/ResponsiveService'
+import { ModalHeader, ModalBody } from 'blockchain-info-components'
+import Tray, { duration } from 'components/Tray'
+import StepIndicator from 'components/StepIndicator'
+import Personal from './Personal'
+// import Address from './Address'
+// import Verify from './Verify'
+
+const HeaderWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  ${media.mobile`
+    flex-direction: column;
+  `};
+`
+
+const stepMap = {
+  [STEPS.personal]: (
+    <FormattedMessage
+      id='modals.identityverification.steps.personal'
+      defaultMessage='Personal'
+    />
+  ),
+  [STEPS.address]: (
+    <FormattedMessage
+      id='modals.identityverification.steps.address'
+      defaultMessage='Address'
+    />
+  ),
+  [STEPS.verify]: (
+    <FormattedMessage
+      id='modals.identityverification.steps.verify'
+      defaultMessage='Verify'
+    />
+  )
+}
+
+export const formName = 'IdentityVerification'
+
+class IdentityVerification extends React.PureComponent {
+  static propTypes = {
+    step: PropTypes.oneOf(keys(STEPS)),
+    isOnfidoEnabled: PropTypes.bool.isRequired,
+    close: PropTypes.func.isRequired
+  }
```

Comment:
We've been setting these after class creation `IdentityVerification.propTypes = ...`

Reply:
ok, will change